.github/workflows/aks-byocni.yaml

```diff
@@ -14,7 +14,7 @@ on:
   #   will disappear from the PR checks: please provide a direct link to the
   #   successful workflow run (can be found from Actions tab) in a comment.
   #
-  # pull_request: {}
+  pull_request: {}
```

Comment:
I think if you want to test `pull_request` then it's better to comment out `pull_request_target`.
Otherwise the result might be unpredictable.

Reply:
looks like commenting out `pull_request_target` made it worse, now it's running with `pull_request_target` trigger 😞 

anyways please review the pr, all the workflows were passing with `pull_request` target before 🚀 🙏